File: enode/src/main/java/org/enodeframework/queue/domainevent/DomainEventHandledMessage.java
package org.enodeframework.queue.domainevent;

import java.io.Serializable;

/**
 * @author anruence@gmail.com
 */
public class DomainEventHandledMessage implements Serializable {
    private String commandId;
    private String aggregateRootId;
    private String commandResult;

    public String getCommandId() {
        return commandId;
    }

    public void setCommandId(String commandId) {
        this.commandId = commandId;
    }

    public String getAggregateRootId() {
        return aggregateRootId;
    }

    public void setAggregateRootId(String aggregateRootId) {
        this.aggregateRootId = aggregateRootId;
    }

    public String getCommandResult() {
        return commandResult;
    }

    public void setCommandResult(String commandResult) {
        this.commandResult = commandResult;
    }
}

File: enode/src/main/java/org/enodeframework/queue/domainevent/DomainEventHandledMessage.kt
package org.enodeframework.queue.domainevent

import org.enodeframework.commanding.CommandReturnType
import org.enodeframework.commanding.CommandStatus
import org.enodeframework.messaging.AbstractReplyMessage

/**
 * @author anruence@gmail.com
 */
class DomainEventHandledMessage : AbstractReplyMessage() {
    override var returnType: CommandReturnType = CommandReturnType.EventHandled
    override var status: CommandStatus = CommandStatus.Success
    override var commandId: String = ""
    override var address: String = ""
    override var aggregateRootId: String = ""
    override var result: String = ""
}
